src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,362 @@
+                  value: "1"
+                - name: CLUSTER_NAME
+                  value: testing-injection
+                - name: COMPUTE_TYPE
+                  value: serverless
+                - name: NRK8S_NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: spec.nodeName
+                - name: NRIA_DISPLAY_NAME
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: spec.nodeName
+                - name: NRIA_CUSTOM_ATTRIBUTES
+                  value: '{"clusterName":"$(CLUSTER_NAME)", "computeType":"$(COMPUTE_TYPE)", "fargateProfile":"[YOUR FARGATE PROFILE]"}'
+                - name: NRIA_PASSTHROUGH_ENVIRONMENT
+                  value: KUBERNETES_SERVICE_HOST,KUBERNETES_SERVICE_PORT,CLUSTER_NAME,CADVISOR_PORT,NRK8S_NODE_NAME,KUBE_STATE_METRICS_URL,KUBE_STATE_METRICS_POD_LABEL,TIMEOUT,ETCD_TLS_SECRET_NAME,ETCD_TLS_SECRET_NAMESPACE,API_SERVER_SECURE_PORT,KUBE_STATE_METRICS_SCHEME,KUBE_STATE_METRICS_PORT,SCHEDULER_ENDPOINT_URL,ETCD_ENDPOINT_URL,CONTROLLER_MANAGER_ENDPOINT_URL,API_SERVER_ENDPOINT_URL,DISABLE_KUBE_STATE_METRICS,DISCOVERY_CACHE_TTL
+                image: newrelic/infrastructure-k8s:2.4.0-unprivileged
+                imagePullPolicy: IfNotPresent
+                resources:
+                  limits:
+                    memory: 300M
+                  requests:
+                    cpu: 100m
+                    memory: 150M
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  readOnlyRootFilesystem: true
+                  runAsUser: 1000
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+                volumeMounts:
+                - mountPath: /var/db/newrelic-infra/data
+                  name: tmpfs-data
+                - mountPath: /var/db/newrelic-infra/user_data
+                  name: tmpfs-user-data
+                - mountPath: /tmp
+                  name: tmpfs-tmp
+                - mountPath: /var/cache/nr-kubernetes
+                  name: tmpfs-cache
+        [...]
+        ```
+      </Collapser>
+    </CollapserGroup>
+
+3. Create a `ClusterRoleBinding`, or add to a previously created one the `ServiceAccount` of the application that is going to be monitored. All the workloads may share the same `ClusterRoleBinding`, but the `ServiceAccount` of each one must be added to it.
+
+    <CollapserGroup>
+      <Collapser
+        id="cluster-role-binding"
+        title="clusterRoleBinding.yaml"
+      >
+      [PLEASE INSERT INSTRUCTIONS ABOUT HOW TO USE THIS SNIPPET. ALSO, I NOTICED IN THE PRIVATE REPO THAT THE FIRST LETTER OF THE YAML FILE (ClusterRoleBinding.yaml) IS CAPITALIZED, UNLIKE OTHER YAML FILE NAMES. IT LOOKS LIKE THE OBJECTS INSIDE THE FILES START WITH CAPITAL LETTERS WHILE THE YAML FILES DO NOT, RIGHT?]
```

Comment:
yes the file name was just a stub and it is not important :)

"Create the following ClusterRoleBinding having as subjects the service account of the pods you want to monitor. Note that there is no need to repeat the same service account twice. Each time you want to monitor a pod with a service account not yet included, you should add it to the list"
I am sure there are better ways to say the same 😅

Reply:
@paologallinaharbur, I inserted this with a few minor changes. Please double-check it.